docs/svo_fps/svo_fps.rst

```diff
@@ -41,28 +41,28 @@ with `~astroquery.svo_fps.SvoFpsClass.get_filter_index`:
     CalibrationReference  object
              Description  object
                 Comments  object
-          WavelengthMean float32   AA
-           WavelengthEff float32   AA
-           WavelengthMin float32   AA
-           WavelengthMax float32   AA
-                WidthEff float32   AA
-           WavelengthCen float32   AA
-         WavelengthPivot float32   AA
-          WavelengthPeak float32   AA
-          WavelengthPhot float32   AA
-                    FWHM float32   AA
+           WavelengthRef float64              AA
+          WavelengthMean float64              AA
+           WavelengthEff float64              AA
+           WavelengthMin float64              AA
+           WavelengthMax float64              AA
+                WidthEff float64              AA
+           WavelengthCen float64              AA
+         WavelengthPivot float64              AA
+          WavelengthPeak float64              AA
+          WavelengthPhot float64              AA
+                    FWHM float64              AA
+                    Fsun float64 erg s / (A cm2)
                PhotCalID  object
                   MagSys  object
-               ZeroPoint float32   Jy
+               ZeroPoint float64              Jy
            ZeroPointUnit  object
-                    Mag0 float32
+                    Mag0 float64
            ZeroPointType  object
-               AsinhSoft float32
+               AsinhSoft float64
         TrasmissionCurve  object
 
-There are options to downselect based on the minimum
-and maximum effective wavelength (``wavelength_eff_min``
-and ``wavelength_eff_max``, respectively).
+.. doctest-skip-all
```

Comment:
why skipping the rest? Instead skipping please refactor so that doctests are passed.


Maybe also add another example with a note that timeout issues are expected, but can be fixed with the passing a large timeout as kwarg.

Reply:
Would you prefer turning on testing for all the code examples to be done in a commit in this pull request or would a separate pull request be more appropriate?

Reply:
Having it here would be fine, after all this is a very short and non-complicated docs.

Reply:
I have turned on remote testing for more code blocks and ensured that the example output matches actual output. The plotting example is still being skipped, but it doesn't execute any `astroquery` code, so I'll leave it the way it is.

Reply:
all plotting examples should be using the `.. plot::` directive at some point, or should be removed, but that's a scope outside of this PR.